SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -985,6 +985,18 @@ class CSolver {
                                    const CConfig *config,
                                    unsigned short val_marker) { }
 
+  /*!
+   * \brief A virtual member.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] numerics - Description of the numerical method.
+   * \param[in] config - Definition of the particular problem.
+   * \param[in] val_marker - Surface marker where the boundary condition is applied.
+   */
+  inline virtual void BC_Velocity(CGeometry *geometry,
+                                   CNumerics *numerics,
```

Comment:
Instead of this can we pass a boolean to BC deforming instead? Just because BC_velocity sounds too much like something a flow solver could have.

Reply:
I've updated this to only have BC_Deforming with a boolean to handle velocity.